recipes/ofeli/all/conanfile.py

```diff
@@ -34,8 +34,6 @@ def validate(self):
         if self.settings.compiler != "gcc":
             raise ConanInvalidConfiguration(
                 "Ofeli is just supported for GCC")
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
```

Comment:
why did you remove this ?

Reply:
i remove it to avoid force cppstd11, but i can add it!